riff-raff/app/ci/S3Build.scala

```diff
@@ -47,6 +64,18 @@ class S3BuildOps(config: Config) extends Logging {
         if (s3BuildEither.isLeft) {
           log.warn(s"Error parsing JSON from $location: $s3BuildEither")
         }
+
+        s3BuildEither match {
+          case Left(_) => {
+            log.warn(s"Error parsing JSON from $location: $s3BuildEither")
```

Comment:
Does this duplicate L65, just above? Should we remove that if statement as well?

I also wonder whether we should log the value in the Left, rather than the Left itself? This is consistent with how it worked previously (and still does on L65), but it might make for clearer logs if we name the `_` on line 69 and log that?

Reply:
🤦🏽 was meant to remove lines 64-66! Good catch!

Reply:
Done.